src/components/bs5/textbox/textInput.scss

```diff
@@ -1,5 +1,5 @@
 .qld-text-input {
-    padding: 12px 16px;
+    padding: 10px 16px;
```

Comment:
Hi @elvishu, can I just confirm the 10px value here. 

Most padding values in this project have a base of 4 (...4px, 8px, 12px, 16px... etc) , so just wanted a quick check please.

Reply:
Hi @stvp-qld, I have reverted the changes on padding-y, from 10px back to 12px.
After discussion with Epinder and Bec, we are keeping the input height as is (52px).